assets/src/editor/events/tickets/editor-ticket/price-calculator/price-modifier-row/price-amount-input.js

```diff
@@ -83,4 +84,12 @@ const PriceAmountInput = ( {
 	] );
 };
 
+PriceAmountInput.propTypes = {
+	prefix: PropTypes.string.isRequired,
+	values: PropTypes.object.isRequired,
+	priceEntity: PropTypes.object.isRequired,
+	priceTypeEntity: PropTypes.object.isRequired,
+	ticketEntity: PropTypes.object.isRequired,
```

Comment:
this one is my fault (sry) :\  

I renamed these props by removing `Entity` from all of them